src/main/java/soot/PackManager.java

```diff
@@ -958,13 +958,11 @@ private void runBodyPacks(SootClass c) {
         // whole shimple or not?
         {
           Body body = m.retrieveActiveBody();
-          if (body instanceof ShimpleBody) {
+          if (body != null && body instanceof ShimpleBody) {
```

Comment:
`!= null` and `null instanceof whatever` is redundant as `null instanceof` resolves to false